tests/TestCodeGen.hs

```diff
@@ -46,6 +49,60 @@ codeGenTests = testGroup "Code generator unit tests"
+swaggerWrapperFormat = testGroup "Swagger Wrapper Format"
+    [ prop @TestProtoWrappers.TestDoubleValue
+           "{\"properties\":{\"wrapper\":{\"format\":\"DoubleValue\",\"type\":\"number\"}},\"type\":\"object\"}"
+           "{\"properties\":{\"wrapper\":{\"format\":\"double\",\"type\":\"number\"}},\"type\":\"object\"}"
+    , prop @TestProtoWrappers.TestFloatValue
+           "{\"properties\":{\"wrapper\":{\"format\":\"FloatValue\",\"type\":\"number\"}},\"type\":\"object\"}"
+           "{\"properties\":{\"wrapper\":{\"format\":\"float\",\"type\":\"number\"}},\"type\":\"object\"}"
+    , prop @TestProtoWrappers.TestInt64Value
+           "{\"properties\":{\"wrapper\":{\"maximum\":9223372036854775807,\"format\":\"Int64Value\",\"minimum\":-9223372036854775808,\"type\":\"integer\"}},\"type\":\"object\"}"
+           "{\"properties\":{\"wrapper\":{\"maximum\":9223372036854775807,\"format\":\"int64\",\"minimum\":-9223372036854775808,\"type\":\"integer\"}},\"type\":\"object\"}"
+    , prop @TestProtoWrappers.TestUInt64Value
+           "{\"properties\":{\"wrapper\":{\"maximum\":18446744073709551615,\"format\":\"UInt64Value\",\"minimum\":0,\"type\":\"integer\"}},\"type\":\"object\"}"
+           "{\"properties\":{\"wrapper\":{\"maximum\":18446744073709551615,\"minimum\":0,\"type\":\"integer\"}},\"type\":\"object\"}"
+    , prop @TestProtoWrappers.TestInt32Value
+           "{\"properties\":{\"wrapper\":{\"maximum\":2147483647,\"format\":\"Int32Value\",\"minimum\":-2147483648,\"type\":\"integer\"}},\"type\":\"object\"}"
+           "{\"properties\":{\"wrapper\":{\"maximum\":2147483647,\"format\":\"int32\",\"minimum\":-2147483648,\"type\":\"integer\"}},\"type\":\"object\"}"
+    , prop @TestProtoWrappers.TestUInt32Value
+           "{\"properties\":{\"wrapper\":{\"maximum\":4294967295,\"format\":\"UInt32Value\",\"minimum\":0,\"type\":\"integer\"}},\"type\":\"object\"}"
+           "{\"properties\":{\"wrapper\":{\"maximum\":4294967295,\"minimum\":0,\"type\":\"integer\"}},\"type\":\"object\"}"
+    , prop @TestProtoWrappers.TestBoolValue
+           "{\"properties\":{\"wrapper\":{\"format\":\"BoolValue\",\"type\":\"boolean\"}},\"type\":\"object\"}"
+           "{\"properties\":{\"wrapper\":{\"type\":\"boolean\"}},\"type\":\"object\"}"
+    , prop @TestProtoWrappers.TestStringValue
+           "{\"properties\":{\"wrapper\":{\"format\":\"StringValue\",\"type\":\"string\"}},\"type\":\"object\"}"
+           "{\"properties\":{\"wrapper\":{\"type\":\"string\"}},\"type\":\"object\"}"
+    , prop @TestProtoWrappers.TestBytesValue
+           "{\"properties\":{\"wrapper\":{\"format\":\"BytesValue\",\"type\":\"string\"}},\"type\":\"object\"}"
+           "{\"properties\":{\"wrapper\":{\"format\":\"byte\",\"type\":\"string\"}},\"type\":\"object\"}"
+    ]
+  where
+    prop ::
+      forall a .
+      (ToSchema a, Typeable a) =>
+      LBS.ByteString ->
+      LBS.ByteString ->
+      TestTree
+    prop wrapperFormat noWrapperFormat =
```

Comment:
What about a slightly clearer name for this, expectSchema? Or something else? If not for the context I have this test would be very unclear

Reply:
Good point; how does 02b0f881f3b3a39b09f6896d7d9cdd9760921748 look?

Reply:
Well you didn't change the definition :)

Reply:
But should be good in that case

Reply:
Hopefully fixed by f3e273a55ff770e47bd4f62b1a42171ae037f162 .

Reply:
CI passed.  Resolving this issue.